src/main/java/com/ega/datarepositorysevice/WebSecurityConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.ega.datarepositorysevice;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+//    @Override
```

Comment:
Do not keep commented code on the repository, if you want to save something, make a commit and tag it, otherwise this becomes clutter when reading the code.